create-publication-from-doi/src/main/java/no/unit/nva/doi/fetch/service/PublicationPersistenceService.java

```diff
@@ -26,13 +28,14 @@
     public static final String WARNING_MESSAGE = "Inserting publication failed.";
     public static final String INSERTING_PUBLICATION_FAILED = WARNING_MESSAGE + "\nAPI-URL:%s\nRequestBody:%s\n";
     private final HttpClient client;
-    private final ObjectMapper objectMapper = MainHandler.createObjectMapper();
+    private final ObjectMapper objectMapper = ObjectMapperConfig.createObjectMapper();
```

Comment:
Do we need this objectMapper here? Can we use the nva commons mapper? I remember we had an issue that the JsonUtils mapper is too strict for when we are importing data from outside. Is this the reason why we use this one here?

Reply:
Yes, we can use the one from nva-commons. I started refactoring without using nva-commons (tried to postpone to other PR) and then I ended up with having to use it.